app/views/persons/index.html.erb

```diff
@@ -47,7 +47,14 @@
     <div class="col-3">
       <% person.groups.each_with_index do |group, index| %>
         <% if group.group_type == "Department" %>
-          <%= group.name %><% unless index == (person.groups.size - 1) %>, <br /><% end %>
+          <%= group.name %><% if group.name == "Learning and Research Services" %>: 
```

Comment:
Hard coding the name of a unit here is probably not  a great idea. Seems like we are missing something in the data model to handle a Parent Group relationship.

Reply:
This was supposed to be a one-off and wouldn't be repeated, but overheard something this morning that leads me to believe otherwise, on other models (namely Services).